src/types.ts

```diff
@@ -1542,8 +1545,10 @@ export type ChannelConfigAutomodThresholds = null | {
 };
 
 export type ChannelConfigFields = {
+  reminders: boolean;
```

Comment:
i think this fails prettier: the properties should be in an alphabetical order

Reply:
Nope ... required vs optional